README.md

```diff
@@ -1,3 +1,11 @@
+# jevons global changes
+
+Forked 31-Aug-2019 to fix some holiday omissions for XASX, XNZE and extend all exchanges back to 1-Jan-1970
```

Comment:
Any chance you could remove these notes from the README?

Reply:
Done now! Sorry. forgot about this one.